src/NBitcoin/Block.cs

```diff
@@ -348,9 +348,6 @@ public void UpdateMerkleRoot()
         /// <returns></returns>
         public bool Check(Consensus consensus)
```

Comment:
I think you can completely delete this whole method

Reply:
Ah yeah good spot, its going away.